### ISSUE TITLE
fix cross compatibility + situations without audio access

### ISSUE THREAD

**Issue by bnidevs** [bnidevs/Meut#2]

**Comment by bnidevs** [comment]
```
self.audio_obj = pyaudio.PyAudio()
self.audio_stream = self.audio_obj.open(format=pyaudio.paInt16,channels=1,rate=12000,input=True,frames_per_buffer=1024)
```

this is the code you're looking to put a try-except around

**Comment by bnidevs** [comment]
need to mandate audio access before starting game

**Comment by bnidevs** [comment]
fixed in 90acc827986a17346ac9be6e7d068e1886ef2e82
